src/main/java/com/conveyal/datatools/editor/jobs/ExportSnapshotToGTFSJob.java

```diff
@@ -74,12 +73,12 @@ public void jobLogic() {
         if (DataManager.useS3) {
             String s3Key = String.format("%s/%s", bucketPrefix, filename);
             try {
-                getDefaultS3Client().putObject(DataManager.feedBucket, s3Key, tempFile);
+                S3Utils.getDefaultS3Client().putObject(S3Utils.DEFAULT_BUCKET, s3Key, tempFile);
             } catch (AmazonServiceException | CheckedAWSException e) {
                 status.fail("Failed to upload file to S3", e);
                 return;
             }
-            LOG.info("Storing snapshot GTFS at s3://{}/{}", DataManager.feedBucket, s3Key);
+            LOG.info("Storing snapshot GTFS at {}", S3Utils.getDefaultBucketUriForKey(s3Key));
```

Comment:
Probably not a huge risk, but using the `getDefaultBucketUriForKey` here feels like it leaves some room for some confusing debugging down the road in the event that the bucket used in the putObject request changed. I don't think the likelihood of that is high (or even medium), just noting that it could be a good place for reusing the same variables. Again, very low risk and probably not worth commenting on!